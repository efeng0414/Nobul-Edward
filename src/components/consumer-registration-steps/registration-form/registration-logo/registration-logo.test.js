import React from "react";
import RegistrationLogo from "./index";

describe("RegistrationLogo Component", () => {
  const componentJSX = <RegistrationLogo />;

  describe("Component Snapshot", () => {
    const componentSnapshot = global.renderer.create(componentJSX).toJSON();

    it("should match stored snapshot", () => {
      expect(componentSnapshot).toMatchSnapshot();
    });
  });
});
