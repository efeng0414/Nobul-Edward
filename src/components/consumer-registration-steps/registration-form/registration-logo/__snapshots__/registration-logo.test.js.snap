// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegistrationLogo Component Component Snapshot should match stored snapshot 1`] = `
<div
  className="consumer-registration-img consumer-registration-img-2"
>
  <img
    className="consumer-registration-logo"
    src={
      Object {
        "default": "file-mock",
      }
    }
  />
</div>
`;
